### PR TITLE
Add accessibility features to illustrations

### PR DESCRIPTION
**Pull request by whmii**

### CODE REVIEW

**Comment by tysongach** [comment]
I believe each of the SVG’s should also have a `role="img"` attribute added to the `svg` element.

Reference: http://www.sitepoint.com/tips-accessible-svg (see Tip 7)

**Comment by whmii** [comment]
haha, @tysongach ya beat me to the punch

**Comment by brendastorer** [comment]
:money_with_wings: :money_with_wings: :money_with_wings:

**Comment by tysongach** [comment]
@whmii Oh, it’s just `role`, rather than `aria-role`.

Why?

4ec89867cc0e1

File: lib/core-001-a.svg [hunk 1]
```diff
@@ -1,5 +1,7 @@
-<svg xmlns="http://www.w3.org/2000/svg"
-  width="160px" height="160px" viewBox="0 0 160 160">
+<svg xmlns="http://www.w3.org/2000/svg" width="160px" height="160px" viewBox="0 0 160 160"
+  aria-role="img" aria-labelledby="title" aria-describedby="desc">
```
Comment (ckundo) [inline lib/core-001-a.svg]: This should be just `role` (see http://www.w3.org/TR/wai-aria/roles#img)

**Comment by whmii** [comment]
@tysongach @ckundo Updated!

**Comment by tysongach** [comment]
This is looking good!

One thing to note: it really depends on how these are used, but in certain (maybe most?) cases, swapping the `role="img"` for `role="presentation"` would be advised.

**Comment by ckundo** [comment]
@tysongach `role='presentation'` may hide content from screen readers. Is that what you want?

**Comment by tysongach** [comment]
@ckundo I think in certain situations, yes. For example, the illustration is just a background or not directly connected to the content on the page.

**Comment by whmii** [comment]
I'd be willing to bet 9 times out of 10 these are going to be used as an image instead of as a background or decorative detiail

**Pull request merged**